mapie/regression.py

```diff
@@ -424,12 +425,12 @@ def _aggregate_with_mask(self, x: NDArray, k: NDArray) -> NDArray:
         Returns:
         --------
         ArrayLike of shape (n_samples_test,)
-            Array of aggregated predictions for each testing  sample.
+            Array of aggregated predictions for each testing sample.
         """
-        if self.cv == "prefit":
+        if self.cv in ["prefit", "split"]:
             raise ValueError(
                 "There should not be aggregation of predictions if cv is "
-                "'prefit'"
+                "'prefit' or 'split'."
```

Comment:
Idem, use a class attribute

Reply:
I propose to use "self.no_agg_cv_" class attribute. I propose also to do the same with "self.no_agg_methods_ = ['naive', 'base']". I am doing these changes.